values.yaml

```diff
@@ -25,6 +25,12 @@ ingress:
   # Name of tls secret to use on ingress
   tlsSecretName: ~
 
+  # Annotations always injected when configuring Ingress ojbect
+  annotations: {}
+
+  # These Annotations will only be injected into the Ingress object if ingress.auth.enabled=true
+  authAnnotations: {}
```

Comment:
And follow up question: if this is empty here and controlled on the API/commander side, why do we need a separate value at all?

Reply:
@ashb we need a separate value because there is an if statement in the chart and if statements don't transfer well to dictionaries.

Reply:
Are both of these dictionaries generated from our nodejs app though?

Reply:
Yea any `.Value`s that are directly consumed by these annotations can probably be removed. All of this logic can be extracted and moved to the `ingress` section of the config generation here: https://github.com/astronomer/houston-api/blob/master/src/lib/deployments/config/index.js#L66-L79.